communitiesuk/digital-form-builder-adapter
# FLS-813: HSRA Eligibility Forms updated

https://mhclgdigital.atlassian.net/browse/FLS-813

### Change description
- Updated eligibility forms
- Added new forms with eligibility question and logic for HSRA redirect to round based on eligibility question



### How to test
- Make sure HSRA VR/RP are live
- Go through the login process
- Click on check eligibility button
- Make sure selecting vacancy register or refurbishment project as answer to question "Why are you applying to this funding?" forwards you to the correct Application tasklist for that round


### Screenshots of UI changes (if applicable)
